scipy-1.0/paper.tex

```diff
@@ -1184,28 +1177,30 @@ \section*{Supporting Information}
-\label{fig:sparse-iter}
-\end{figure}
-
-\begin{figure}[H]
-\centering
-\includegraphics[width=\textwidth]{supporting_info/asv_bench/sparse/sparse_coo_dia_to_csr_csc}
-\caption{Airspeed velocity benchmarks for SciPy COO / DIA to CSR / CSC matrix format conversion over roughly three years of commits prior to the release of version 1.0. Python version 2.7, Cython version 0.27.3, and NumPy 1.8.2 were used. The values were obtained from our regularly-updated suite of performance results available online at \url{http://pv.github.io/scipy-bench/}.}
-\label{fig:sparse-conv}
-\end{figure}
-
-\section{Polynomial Interpolators: Additional Details}
+%
+%\begin{figure}[H]
+%\centering
+%\includegraphics[width=0.8\textwidth]{supporting_info/knn_complexity_confirm}
+%\caption{Assessment of \texttt{cKDTree} nearest neighbor query time complexity as a function of the number of input points (N) along with similar assessment for related algorithms in scikit-learn. The calculations are based on code previously reported\cite{knn-jake}, but updated to use SciPy 1.0.0, Python 3.6, NumPy 1.13.1, and Cython 0.27.3.}
+%\label{fig:knn-complexity}
+%\end{figure}
+%
+%\begin{figure}[H]
+%\centering
+%\includegraphics[width=\textwidth]{supporting_info/asv_bench/sparse/sparse_iteration_bench}
+%\caption{Airspeed velocity benchmarks for SciPy CSC and CSR matrix row iteration over roughly three years of commits prior to the release of version 1.0. Python version 2.7, Cython version 0.27.3, NumPy 1.8.2, and the indicated non-zero element density fractions were used. The partial performance regression observed shortly after the 1.0 release is likely related to a correctness fix\cite{sparse-regress}. The values were obtained from our regularly-updated suite of performance results available online at \url{http://pv.github.io/scipy-bench/}.}
+%\label{fig:sparse-iter}
+%\end{figure}
+%
+%\begin{figure}[H]
+%\centering
+%\includegraphics[width=\textwidth]{supporting_info/asv_bench/sparse/sparse_coo_dia_to_csr_csc}
+%\caption{Airspeed velocity benchmarks for SciPy COO / DIA to CSR / CSC matrix format conversion over roughly three years of commits prior to the release of version 1.0. Python version 2.7, Cython version 0.27.3, and NumPy 1.8.2 were used. The values were obtained from our regularly-updated suite of performance results available online at \url{http://pv.github.io/scipy-bench/}.}
+%\label{fig:sparse-conv}
+%\end{figure}
+
+\subsection*{Polynomial Interpolators: Additional Details}
```

Comment:
Should this be kept?

Reply:
Actually not too useful probably, could be cut.